src/screens/FeedScreen/index.js

```diff
@@ -248,7 +241,6 @@ const FeedScreen = (props) => {
         ref={listRef}
         contentHeight={dimen.size.FEED_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
         data={feeds}
-        onEndReach={onEndReach}
         onRefresh={onRefresh}
         onScroll={handleScrollEvent}
         onScrollBeginDrag={handleOnScrollBeginDrag}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been removed from the `TiktokScroll` component. This is consistent with the removal of the `onEndReach` function earlier in the code.
```diff
-        onEndReach={onEndReach}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->